frontend/src/pages/site/components/about/about.tsx

```diff
@@ -1,25 +1,82 @@
-import { type SiteAboutContent } from '~/packages/sections/sections.js';
+import { Input } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useAppForm } from '~/libs/hooks/hooks.js';
+import {
+  type SiteAboutContent,
+  type SiteAboutUpdateContentDto,
+} from '~/packages/sections/sections.js';
+import { siteAboutUpdateContentValidationSchema } from '~/packages/sections/sections.js';
+import { Overlay } from '~/pages/site/components/overlay/overlay.js';
+import { useSectionUpdate } from '~/pages/site/libs/hooks/hooks.js';
 
+import { Textarea } from './components/components.js';
```

Comment:
```suggestion
```
it should be part of a common `Input` component. Add new prop "rows" if we provide this prop - then `textarea` should be rendered. if no - `input`